dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -47,54 +51,57 @@
 import com.google.common.truth.Truth;
 
 public class DashboardTest {
-  
+
   private static Path outputDirectory;
   private Builder builder = new Builder();
-  
+
   @BeforeClass
-  public static void setUp() throws ArtifactDescriptorException, IOException, TemplateException {
-    outputDirectory = DashboardMain.generate();
+  public static void setUp() throws IOException {
+    outputDirectory = Files.createDirectories(Paths.get("target", "dashboard"));
   }
-  
+
   @AfterClass
   public static void cleanUp() throws IOException {
     MoreFiles.deleteRecursively(outputDirectory, RecursiveDeleteOption.ALLOW_INSECURE);
   }
 
   @Test
   public void testMain() throws IOException, TemplateException, ArtifactDescriptorException {
+    // Ensuring normal execution doesn't cause any exception
     DashboardMain.main(null);
   }
-  
+
   @Test
   public void testDashboard()
       throws IOException, TemplateException, ParsingException, ArtifactDescriptorException {
-    
+    // Creates "dashboard.html" in outputDirectory
+    outputDirectory = DashboardMain.generate();
```

Comment:
I think you've introduced order dependence into the tests. This isn't guaranteed to work. I suggest leaving this test alone and moving your test to a new class.